docs/erp_notification.adoc

```diff
@@ -10,44 +10,83 @@
 :toc: macro
 :toclevels: 3
 :toc-title: Inhaltsverzeichnis
+:toclevels: 1
```

Comment:
doppelt aus Zeile 11?

Reply:
Wir haben in unseren .source Dateien eine vorgefertigte config. Ich habe noch mal `toclevels: 1` eingefügt, um das zu überschreiben, da es hier dann übersichtlicher ist

Reply:
Danke für das Review